lib/src/main/java/com/thenewboston/api/common/GetDataSource.kt

```diff
@@ -38,6 +40,12 @@ class GetDataSource @Inject constructor(private val networkClient: NetworkClient
         return Outcome.Success(result)
     }
 
+    suspend fun primaryValidatorDetails(): Outcome<PrimaryValidatorDetails> {
+        val result = networkClient.defaultClient.get<PrimaryValidatorDetails>(PrimaryValidatorAPIEndpoints.CONFIG_ENDPOINT)
+
+        return Outcome.Success(result)
+    }
```

Comment:
Can we add test cases for this?

Reply:
The PrimaryValidatorDetails test fails in `GetDataSourceTest` because the mock engine used is `BankApiMockEngine`
How do I get around this?
Should I create a `PrimaryApiMockEngine`

Reply:
Yes since the response is slightly different in the primary validator api for that endpoint, we can create a separate mock engine for that api.